client/tests/webdriver/pages/page.js

```diff
@@ -0,0 +1,16 @@
+const DEFAULT_CREDENTIALS = { user: 'erin', superUser: 'arthur' }
+
+class Page {
+	open(pathName = '/', credentials = DEFAULT_CREDENTIALS.user) {
+		const urlToGet = `${browser.options.baseUrl}${pathName}?user=${credentials}&pass=${credentials}`
+		browser.url(urlToGet)
+	}
+
+	openAsSuperUser(pathName = '/') {
+		const credentials = 'arthur'
```

Comment:
Shouldn't this use `DEFAULT_CREDENTIALS.superUser`?

Reply:
Done.